tests/test_integration/stencil_definitions.py

```diff
@@ -27,23 +27,22 @@
 
 
 REGISTRY = gt_utils.Registry()
-EXTERNALS_REGISTRY = gt_utils.Registry()
+PROPERTY_REGISTRY = gt_utils.Registry()
 
 
-def register(externals):
-    if callable(externals):
-        func = externals  # wacky hacky!
-        EXTERNALS_REGISTRY.setdefault(func.__name__, {})
+def register(definition=None, *, externals=None, splitters=None):
+    properties = {}
+    if externals:
+        properties["externals"] = externals
```

Comment:
I think this would be safer:
```suggestion
        properties["externals"] = externals or {}
```